app/controllers/concerns/hyku_addons/catalog_controller_behavior.rb

```diff
@@ -15,8 +15,8 @@ module CatalogControllerBehavior
         config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collections'
         config.add_facet_field solr_name("institution", :facetable), limit: 5, label: 'Institution'
         config.add_facet_field solr_name("language", :facetable), limit: 5, label: 'Language'
-        config.add_facet_field solr_name("org_unit", :facetable), limit: 5, label: 'Department'
-        config.add_facet_field solr_name("audience", :facetable), limit: 5, label: 'OER Audience', if: proc { |context, _config, _opts| context.current_account&.settings&.dig("locale_name") == "redlands" }
+        config.add_facet_field solr_name("org_unit_sim", :facetable), limit: 5, label: 'Department'
+        config.add_facet_field solr_name("audience_sim", :facetable), limit: 5, label: 'OER Audience', if: proc { |context, _config, _opts| context.current_account&.settings&.dig("locale_name") == "redlands" }
```

Comment:
Let's just hard code it because it is clearer and saves time.
```suggestion
        config.add_facet_field "org_unit_tesim", limit: 5, label: 'Department'
        config.add_facet_field "audience_tesim", limit: 5, label: 'OER Audience', if: proc { |context, _config, _opts| context.current_account&.settings&.dig("locale_name") == "redlands" }
```

Reply:
Ohhh I see what you meant. Ok